.changeset/chilled-zebras-help.md

```diff
@@ -0,0 +1,19 @@
+---
+"@neo4j/graphql": patch
+---
+
+Deprecate single element relationships:
```

Comment:
I think this changeset is by far not long enough - this needs a lot of explanation as to the rationale IMO!

Reply:
Isn't that rationally better left for the migration guide?